backend/spec/features/admin/products/edit/taxons_spec.rb

```diff
@@ -26,13 +26,16 @@ def assert_selected_taxons(taxons)
       visit spree.edit_admin_product_path(product)
 
       assert_selected_taxons([taxon_1])
-
       select2_search "Clothing", from: "Taxon"
       assert_selected_taxons([taxon_1, taxon_2])
 
+      find('.edit_product', visible: true, obscured: false).click
       click_button "Update"
+      expect(product.taxons.reload.count).to be 2
 
-      expect(find(".flash")).to have_text "Product \"#{product.name}\" has been successfully updated!"
+      within('.flash') do
+        expect(page).to have_content("Product \"#{product.name}\" has been successfully updated!")
```

Comment:
What about taking advantage of this change in order to improve string interpolation avoiding escaping double quotes? I was thinking something like that:

```ruby
%(Product "#{product.name}" has been successfully updated!)
```